openedx-devstack-ubuntu/LICENSE

```diff
@@ -0,0 +1,9 @@
+The MIT License (MIT)
```

Comment:
I don't think a license per template is needed. The repo has a MIT license as well.